src/demo/share/jfc/J2Ddemo/java2d/Intro.java

```diff
@@ -443,7 +443,10 @@ public void run() {
             }
 
             while (thread == me) {
-                Scene scene = director.get(index);
+              if (index == 5) {
+                  System.exit(0);
+              }
+              Scene scene = director.get(index);
```

Comment:
Leftover debugging code?

Reply:
Yes, it is. I wanted a way to quit Java2Demo at the exact same point. I will revert it.